File: Basic Coding Problems/02 If - Elses/031 Write a program to input month number and print number of days in that month/Java/number_of_days_in_month.java
import java.util.Scanner;

public class number_of_days_in_month {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the month number: ");
        int month = scanner.nextInt();

        if (month == 1 || month == 3 || month == 5 || month == 7 || month == 8 || month == 10 || month == 12) {
            System.out.println("The number of days in this month is 31.");
        } else if (month == 4 || month == 6 || month == 9 || month == 11) {
            System.out.println("The number of days in this month is 30.");
        } else if (month == 2) {
            System.out.println("The number of days in this month is 28 or 29.");
        } else {
            System.out.println("Invalid month number.");
        }
    }
}

File: Basic Coding Problems/02 If - Elses/031 Write a program to input month number and print number of days in that month/C++/number_of_days_in_month.cpp
#include <iostream>

using namespace std;

int main() {
    int month;

    cout << "Enter the month number: ";
    cin >> month;

    if (month == 1 || month == 3 || month == 5 || month == 7 || month == 8 || month == 10 || month == 12) {
        cout << "The number of days in this month is 31.\n";
    } else if (month == 4 || month == 6 || month == 9 || month == 11) {
        cout << "The number of days in this month is 30.\n";
    } else if (month == 2) {
        cout << "The number of days in this month is 28 or 29.\n";
    } else {
        cout << "Invalid month number.\n";
    }

    return 0;
}